### PR TITLE
[ticket/16477] Restore .icon class selector in icons.css

### PR DESCRIPTION
**Pull request by rxu**
**For the master branch only**.

The line was added in #5957 but is missing from the master branch.

Checklist:

- [x] Correct branch: master for new features; 3.3.x & 3.2.x for fixes
- [x] Tests pass
- [x] Code follows coding guidelines: [master](https://area51.phpbb.com/docs/dev/master/development/coding_guidelines.html), [3.3.x](https://area51.phpbb.com/docs/dev/3.3.x/development/coding_guidelines.html) and [3.2.x](https://area51.phpbb.com/docs/dev/3.2.x/development/coding_guidelines.html)
- [x] Commit follows commit message [format](https://area51.phpbb.com/docs/dev/3.3.x/development/git.html)

<a href="https://tracker.phpbb.com/browse/PHPBB3-16477">PHPBB3-16477</a>.

### CODE REVIEW

**Comment by rxu** [comment]
!set Styles

**Comment by mrgoldy** [comment]
I tripled checked all the css changes in master, while resolving all the conflicts, yet still managed to miss this one. My apoligies.

File: phpBB/styles/prosilver/theme/icons.css [hunk 1]
```diff
@@ -27,6 +27,8 @@ blockquote cite:before,
 	text-rendering: auto; /* optimizelegibility throws things off #1094 */
 }
 
+.icon { padding-right: 2px; }
```
Comment (mrgoldy) [inline phpBB/styles/prosilver/theme/icons.css:30]: Oh, if you could change it to not be on one line, then I can merge it.
```suggestion
.icon { 
	padding-right: 2px;
}
```

**Comment by hanakin** [comment]
this is not going to matter as all icon classes will be removed from master going forward once the icon rework is merged

**Comment by rxu** [comment]
Can't wait for it got merged 😃 
Currently it's just inconsistent and looks not beautiful, but whatever you decide.

**Pull request merged**